src/routes/safe/components/Apps/ManageApps.js

```diff
@@ -103,20 +107,38 @@ const ManageApps = ({ appList, onAppAdded, onAppToggle }: Props) => {
     return exists ? 'This app is already registered.' : undefined
   }
 
-  const onFormStatusChange = ({ pristine, valid, validating }) => {
-    if (!pristine) {
-      setIsSubmitDisabled(validating || !valid || appInfo.error)
+  const safeAppValidator = async (value: string) => {
```

Comment:
This doesn't have to be defined inside a component

Reply:
yes it does, it uses `uniqueAppValidator` that uses `appList`